src/Aspire.Dashboard/Components/Controls/LogViewer.razor.cs

```diff
@@ -74,7 +76,7 @@ private void OnBrowserResize(object? o, EventArgs args)
 
     private string GetDisplayTimestamp(DateTimeOffset timestamp)
     {
-        var date = _convertTimestampsFromUtc ? TimeProvider.ToLocal(timestamp) : timestamp.DateTime;
+        var date = IsTimestampUtc ? timestamp.DateTime : TimeProvider.ToLocal(timestamp);
 
         return date.ToString(KnownFormats.ConsoleLogsUITimestampFormat, CultureInfo.InvariantCulture);
```

Comment:
I think we should include `Z` suffix on timestamps in UTC mode. That makes it clear in the UI that these are UTC timestamps.

Don't add the local UTC offset in local mode. That would be a lot of extra noise.

Will need a timestamp format for local (current one) and one for UTC that adds https://learn.microsoft.com/en-us/dotnet/standard/base-types/custom-date-and-time-format-strings#time-zone-k-format-specifier

Reply:
Just to confirm, this is what you're looking for?

local:
![image](https://github.com/user-attachments/assets/3c28d575-dbe8-401b-8aaf-a0ec9578d5e4)

utc:
![image](https://github.com/user-attachments/assets/6880c5a3-d0aa-40a9-93c9-4880fb04ee63)